broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/NewPartitionTransitionImpl.java

```diff
@@ -29,25 +28,17 @@
   private PartitionTransitionContext context;
   private ConcurrencyControl concurrencyControl;
   private PartitionTransitionProcess lastTransition;
-  // these two should be set/cleared in tandem
+
+  // transient state - to keep track of the current transitions
   private PartitionTransitionProcess currentTransition;
```

Comment:
:thought_balloon: Not sure current transition is the right name anymore, because it could also refer to a transition that will happen in the future. At the same time, I don't have a better name off the top of my head

Reply:
currentTailTransition ? :thinking: :sweat_smile: